x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -172,18 +172,28 @@ export const APP_HOST_ISOLATION_EXCEPTIONS_PATH =
   `${APP_PATH}${HOST_ISOLATION_EXCEPTIONS_PATH}` as const;
 export const APP_BLOCKLIST_PATH = `${APP_PATH}${BLOCKLIST_PATH}` as const;
 
+// cloud logs to exclude from default index pattern
+export const EXCLUDE_ELASTIC_CLOUD_INDICES = [
+  '-*elastic-cloud-logs-*',
+  '-.ds-logs-elastic_agent*',
+  '-.ds-logs-system-*',
```

Comment:
Consider removing `-.ds-logs-elastic_agent*` and `-.ds-logs-system-*` from the default excludes because it's a breaking-change for users who haven't modified the default `securitySolution:defaultIndex` setting